wpilibNewCommands/.styleguide

```diff
@@ -1,8 +1,42 @@
+cppHeaderFileInclude {
+  \.h$
+  \.hpp$
+  \.inc$
+}
+
+cppSrcFileInclude {
+  \.cpp$
+}
+
+generatedFileExclude {
```

Comment:
Isn't this unnecessary at this level?

Reply:
I suppose. Some parts of the root .styleguide are out of date as well.